ci/scenarios/vPool/add_remove_vpool_concurrently/main.py

```diff
@@ -88,10 +87,13 @@ def remove_vpools(self):
         self.LOGGER.info("Starting to validate removal of vpools concurrently.")
         dels = {}
         for vpool in self.vpools:
-            dels[vpool.guid] = self.api.post(api='vpools/{0}/shrink_vpool'.format(vpool.guid), data=json.dumps({'storagerouter_guid': vpool.storagedrivers[0].storagerouter.guid}))
-        for vpool_name, task in dels.iteritems():
-            self.LOGGER.info('-> deleting vpool nr {0}: {1}'.format(vpool_name, self.api.wait_for_task(task)[0]))
-        self.LOGGER.info("Concurrent removal of vpools finished.")
+            for storagedriver in vpool.storagedrivers:
+                dels[vpool.guid] = self.api.post(api='vpools/{0}/shrink_vpool'.format(vpool.guid), data=json.dumps({'storagerouter_guid': storagedriver.storagerouter.guid}))
```

Comment:
Use `storagedriver.storagerouter_guid` instead
Identical result, but more performant like this